new-site/src/components/Hero.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react';
+import { withPrefix } from 'gatsby';
+import { Koros } from 'hds-react';
+import PropTypes from 'prop-types';
+
+import './Hero.scss';
+
+const Hero = ({ children, backgroundImageUrl, title, text }) => {
+  return (
+    <div className="hero-container">
+      <div className="hero-wrapper">
+        <div className="hero">
+          <div className="hero-content">
+            <div className="hero-content-text">
```

Comment:
The class `hero-content-text` is not needed anymore, can be deleted from html too.

Reply:
- [x] remove unnecessary wrapper